eng/scripts/TypeSpec-Compare-CurrentToCodegeneration.ps1

```diff
@@ -82,7 +82,8 @@ Verify no diff
 "
 
 # prevent warning related to EOL differences which triggers an exception for some reason
-git -c core.safecrlf=false diff --ignore-space-at-eol --exit-code -- "*.java" ":(exclude)**/src/test/**" ":(exclude)**/src/samples/**" ":(exclude)**/src/main/**/implementation/**"
+git -c core.safecrlf=false diff --ignore-space-at-eol --exit-code -- "*.java" ":(exclude)**/src/test/**" ":
+(exclude)**/src/samples/**" ":(exclude)**/src/main/**/implementation/**" ":(exclude)**/src/main/**/resourcemanager/**/*Manager.java"
```

Comment:
We doesn't pass package-version to `tsp-update`, making it unavailable to codegen. 
Since we need package-version for codegen, possible solutions are:
1. Ignore `XXManager` diff here(as this workaround does), and also ignore diff in sync-sdk.py from autorest.java(which will also call `tsp-client update`). I don't see much issue, as XXManager is kinda stable.
2. In script, use current version from `version_client.txt` as package-version and pass it to to `tsp-update`, e.g. `tsp-update --emitter-options "package-version=1.1.0"`. Tested it works.
3. In codegen, deduce current version from `version_client.txt`, and use that as package-version when missing

Reply:
Or make it like data-plane, that name/version is read from a properties file in resources.

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/devcenter/azure-developer-devcenter/src/main/java/com/azure/developer/devcenter/DevBoxesClientBuilder.java#L275-L278

https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/devcenter/azure-developer-devcenter/src/main/resources/azure-developer-devcenter.properties

It seems mgmt can also use that. This way, the version (name too) won't appear in code.

Reply:
Thanks, that's a better option to me. Created issue: https://github.com/Azure/autorest.java/issues/3020